extensions/wikia/AdEngine/js/provider/directGpt.js

```diff
@@ -23,7 +23,17 @@ define('ext.wikia.adEngine.provider.directGpt', [
 ) {
 	'use strict';
 
-	var context = adContext.getContext();
+	var context = adContext.getContext(),
+		sraEnabled = !context.opts.disableSRA,
```

Comment:
`sraEnabled` flag simply turns on btfBlocker. What about keeping sraEnabled always set to true and manipulate which slots belong to ATF? It will disable SRA built from TLB/MR/SKIN and keep BTF delayed